integration-tests/sql/src/test/java/org/apache/camel/quarkus/component/sql/it/SqlTest.java

```diff
@@ -41,6 +42,9 @@
 @QuarkusTestResource(DerbyTestResource.class)
 class SqlTest {
 
+    @ConfigProperty(name = "quarkus.datasource.db-kind")
```

Comment:
```
org.junit.platform.commons.JUnitException: @ConfigProperty is not supported in @QuarkusIntegrationTest tests. Offending field is org.apache.camel.quarkus.component.sql.it.SqlTest.dbKind
```

Maybe try resolving the db type by invoking a REST endpoint or something like that.

Reply:
It worked for me localy, so I thought that it is solved in the newest Qarkus -> using a rest is the solution. I'll change that

Reply:
fixed